.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
+  "endOfLine": "lf",
+  "semi": false,
```

Comment:
most of the current code has semicolons at the end. I would set it to true.

Reply:
Ok, thanks for the link to the related discussion. I don't want to have a long discussion on comma vs semi-colon, since it might never end :-). I'm ok to have semi-colon instead of commas.